raiden/ui/runners.py

```diff
@@ -166,6 +166,20 @@ def _start_services(self):
             app_.raiden.chain.client.web3,
         ))
 
+        if (
```

Comment:
nitpick: Adjust accordingly to make the multiline if a bit more readable as I explained in my previous comment